tap/tlstapper/bpf/include/maps.h

```diff
@@ -11,9 +11,10 @@ Copyright (C) UP9 Inc.
 #define FLAGS_IS_READ_BIT (1 << 1)
 
 #define CHUNK_SIZE (1 << 12)
-#define CHUNK_X8_SIZE (1 << 15)
 #define MAX_CHUNKS_PER_OPERATION (8)
 
+#define SSL_INFO_MAX_TTL_NANO (1000000000l * 60l)
```

Comment:
How did you come to choose this value?

Reply:
Its a minute in nano, it seems logical to wait maximum a minute between two ssl_read/write calls of the same message.